### PR TITLE
chore(docs): compare PRs against next branch instead of current

### PR DESCRIPTION
**Pull request by binoy14**
### Description

- fixes token mismatch when deleting dataset
- better message when no changes to documentation coverage

### What to review

The actions conditionals are valid

### Notes for release

N/A

### CODE REVIEW

File: .github/workflows/docReport.yml [hunk 1]
```diff
@@ -40,11 +40,11 @@ jobs:
         env:
           NODE_OPTIONS: --max_old_space_size=8192
 
-      - name: Create Docs Report on current
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/current' }}
+      - name: Create Docs Report on current or next
+        if: ${{ github.event_name == 'push' && (github.ref == 'refs/heads/current' || github.ref == 'refs/heads/next') }}
```
Comment (binoy14) [inline .github/workflows/docReport.yml:44]: Not sure if the `event_name` is still push when a PR is merged. Does anyone know?

File: .github/workflows/docReport.yml [hunk 2]
```diff
@@ -40,11 +40,11 @@ jobs:
...
         env:
           DOCS_REPORT_TOKEN: ${{ secrets.DOCS_REPORT_DATASET_TOKEN }}
-          DOCS_REPORT_DATASET: production
+          DOCS_REPORT_DATASET: ${{ github.ref == 'refs/heads/current' && production || next}}
```
Comment (binoy14) [inline .github/workflows/docReport.yml:47]: Also is this syntax valid?

File: .github/workflows/docReport.yml (hunk 1 above)
Comment (bjoerge) [reply .github/workflows/docReport.yml:44]: Seems fair to assume so, esp. since there doesn't seem to be a `merge` event.

**Review by bjoerge — APPROVED** [review]
Nice!

**Comment by binoy14** [comment]
oh just realized the `next` branch is not in the allowed list for the workflow to run. Will add it

File: .github/workflows/docReport.yml (hunk 1 above)
Comment (binoy14) [reply .github/workflows/docReport.yml:44]: It might be trial and error. I will merge and check

**Pull request merged**